src/plugins/codeStyle/index.ts

```diff
@@ -36,6 +36,30 @@ export default class CodeStyle {
         const requireConditionGroup = conditionStyle === 'group';
         const validateAAStyle = aaCommaStyle !== 'off';
         const walkExpressions = validateAAStyle;
+        const validateEolLast = eolLast !== 'off';
+        const disallowEolLast = eolLast === 'never';
+
+        // Check if the file is going backwards from the last token.
+        let isFileEmpty = true;
+        for (let i = file.parser.tokens.length - 1; i >= 0; i--) {
```

Comment:
Use a `const { tokens } = file.parser;` once to make code less verbose and more efficient.

Reply:
Done by 95a4f6e